key.go

```diff
@@ -0,0 +1,69 @@
+package statter
+
+import (
+	"sync"
+	"unsafe"
+)
+
+var keyPool = sync.Pool{
+	New: func() interface{} {
+		return &key{b: make([]byte, 0, 256)}
+	},
+}
+
+type key struct {
+	b []byte
+}
```

Comment:
What's the point of key not being a type alias to `[]byte`? I feel like it would make the rest of the code more readable tbh.

Reply:
making `key` a `[]byte` alias makes it alloc. I have not entirely looking into why yet.